### DISCUSSION TITLE
How to undo "allow selected" for browser extension?

### DISCUSSION THREAD

**Discussion by juxeii** [keepassxreboot/keepassxc#10577] [Q&A] [answered]
Hi,

I am very new to keepassxc and I cannot find out how revert some permissions I have given to some sites.
KeepassXC asks for `Allow Selected` in the `Browse Access Request`.
Now, if I have accidently given access to an entry, how can I later remove this permission again?

**Comment by droidmonkey** [comment, marked as answer]
If you didn't check the "remember" checkbox, then nothing happened. You'll be asked again. If you did check the remember checkbox, you'll have to go to the entry, double click or press enter to open for editing, go to the properties tab, select the browser custom data and click remove. Press ok and save changes.

**Reply by theking2** [reply]
This would be great to include in the documentation . https://keepassxc.org/docs/KeePassXC_UserGuide

**Comment by juxeii** [comment]
When I go to the properties tab I don't see browser custom data. I only see some fields 
```
Created
Modified
Accessed
Uuid

```
After that, there is only `Plugin Data`, which is an empty table.

Edit: I can see browser custom data for other entries. But why is the entry in question still showing up as suggestion in the browser extension? I have to remove it in order to have the one and only entry for this site.

Edit2: Nevermind, I had multiple similar entries. Now I found the correct one. Thx for your help!
